File: 백준/Silver/2563. 색종이/색종이.java
import java.util.Scanner;

public class Main {
	static int[][] check = new int [105][105];
	static int N, ans;

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		N = sc.nextInt();
		while (N-- > 0) {
			int a = sc.nextInt(), b = sc.nextInt();
			for (int i = a; i < a + 10; i++) {
				for (int j = b; j < b + 10; j++) {
					if (check[i][j] == 0) {
						check[i][j] = 1;
						ans++;
					}

				}
			}
		}
		System.out.println(ans);
	}

}

File: 백준/Silver/2563. 색종이/색종이.cc
#include<iostream>
using namespace std;
int check[105][105];
int main() {
	int n, ans = 0;
	cin >> n;
	for (int i = 0; i < n; i++) {
		int a, b;
		cin >> a >> b;
		for (int j = a; j < a + 10; j++) {
			for (int k = b; k < b + 10; k++) {
				if (!check[j][k]) {
					check[j][k] = 1;
					ans++;
				}
			}
		}
	}
	cout << ans;
}